### ISSUE TITLE
Variable not in scope: d1 :: t0 -> t 

### ISSUE THREAD

**Issue by ohase** [tidalcycles/atom-tidalcycles#82]
Variable not in scope: d1 :: t0 -> t

    • Variable not in scope: sound :: [Char] -> t0
    • Perhaps you meant ‘round’ (imported from Prelude)

**Comment by ndr-brt** [comment]
what's the command that caused this error?

**Comment by yaxu** [comment]
My guess is that the tidal library wasn't installed properly. Try `cabal v1-install tidal` from a terminal window.

**Comment by leidundelend** [comment]
I got the same error and tested the tidal installation in ghci. It works fine. When I downgrade the atom package to 3.8.2 everything works as expected. But in 3.8.3 and 3.8.4 it seems to not load the boot file correctly: when I execute the (important) commands from the boot file manually I can make it work...

**Comment by ndr-brt** [comment]
this issue went reported when the 3.8.3 were not yet released, don't think it's the same problem

**Comment by k0nfus1us** [comment]
I got the same error; ghci + tidal (3.8.4) via CLI working fine.
Running this on Windows right now, and I suspect the ghci path might be parsed wrong somehow.
Defining the `ghciPath:` parameter in the config.cson (or via UI->Settings...) manually yields a bit more information in the console output: 
```
Custom ghci path configured: C:\Program Files\Haskell Platform\8.4.3\bin\
Ghci command: C:\Program Files\Haskell Platform\8.4.3\bin\ghci
Ghc-pkg command: C:\Program Files\Haskell Platform\8.4.3\bin\ghc-pkg
Load BootTidal.hs from C:\Users\SomeUser\.atom\packages\tidalcycles\lib\BootTidal.hs
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
```
And of course `'C:\Program'` is not recognized as a valid command...

**Comment by ndr-brt** [comment]
@k0nfus1us check out v3.8.6, should be fixed

**Comment by k0nfus1us** [comment]
@ndr-brt I updated the package in Atom to 3.8.6 and restarted, ghci now showed as loaded! But before parsing the `BootTidal.hs` it spit out another error: `syntax:  :script `
It turns out the path to `BootTidal.hs` needs to be free of spaces as well! Once I copied the tidalcycles folder to a path without spaces and set this path in atom-tidalcycles the boot went through.

**Comment by ndr-brt** [comment]
woa! will fix this too! thanks for reporting

**Comment by ndr-brt** [comment]
there is a problem... ghci's ":script" command does not support `"` and `'` to fold the path

**Comment by ndr-brt** [comment]
This issue is resolved, opened #95 for the space-in-path problem
